Podcast/SearchHeaderView.swift

```diff
@@ -21,9 +21,9 @@ enum searchHeaderViewType {
     var title: NSMutableAttributedString {
         switch self {
         case .itunes:
-            let attributedString = NSMutableAttributedString(string: "Can’t find a series you’re looking for? You can now search iTunes directly.")
-            attributedString.addAttribute(.foregroundColor, value: UIColor.sea, range: NSRange(location: 52, length: 13))
-            attributedString.addAttribute(.foregroundColor, value: UIColor.slateGrey, range: NSRange(location: 66, length: 9))
+            let attributedString = NSMutableAttributedString(string: "Can’t find a series you’re looking for? You can now search our database directly.")
```

Comment:
hmm .. wording seems a bit weird still .. Maybe `Can’t find a series you’re looking for? Search the web to help us add your series to our collection`